packages/kbn-reporting/server/types.ts

```diff
@@ -38,6 +43,7 @@ export type CreateJobFn<JobParamsType = BaseParams, JobPayloadType = BasePayload
 export type RunTaskFn<TaskPayloadType = BasePayload> = (
   jobId: string,
   payload: TaskPayloadType,
+  taskInstanceFields: TaskInstanceFields,
```

Comment:
To make the task lifecycle timing information available to the run task functions, I just added this new parameter to for all Run Task functions.